x-pack/plugins/security/server/lib/authorization/actions/saved_object.ts

```diff
@@ -18,6 +26,6 @@ export class SavedObjectActions {
       throw new Error('operation is required and must be a string');
```

Comment:
Not part of this PR, but since I noticed it:

```suggestion
      throw new Error('type is required and must be a string');
```